kernel/src/actions/mod.rs

```diff
@@ -563,6 +576,26 @@ pub(crate) struct SetTransaction {
     pub(crate) last_updated: Option<i64>,
 }
 
+impl SetTransaction {
+    pub(crate) fn new(app_id: String, version: i64, last_updated: Option<i64>) -> Self {
+        Self {
+            app_id,
+            version,
+            last_updated,
+        }
+    }
+
+    pub(crate) fn into_engine_data(self, engine: &dyn Engine) -> DeltaResult<Box<dyn EngineData>> {
+        let values = [
+            self.app_id.into(),
+            self.version.into(),
+            Scalar::Null(DataType::LONG),
```

Comment:
Seems incorrect to hard-wire NULL for a field? Why even declare the field in that case?
```suggestion
            self.last_updated.map_or(Scalar::Null(DataType::LONG), |u| u.into()),
```

Reply:
(and yea sorry this was an old vestige that I forgot to update)